src/gui/SetupDialog.cpp

```diff
@@ -249,6 +251,15 @@ SetupDialog::SetupDialog( ConfigTabs _tab_to_open ) :
 	addLedCheckBox("Reopen last project on start",
 		m_openLastProject, SLOT(toggleOpenLastProject(bool)));
 
+	LedCheckBox * sideBarOnRight = new LedCheckBox(
+				tr( "Sidebar on right" ),
+								misc_tw );
+	labelNumber++;
+	sideBarOnRight->move( XDelta, YDelta*labelNumber );
+	sideBarOnRight->setChecked( m_sideBarOnRight );
+	connect ( sideBarOnRight, SIGNAL( toggled( bool ) ),
+				this, SLOT( toggleSideBarOnRight( bool ) ) );
```

Comment:
As there's `addLedCheckBox` function, you can do it in two lines:
```cpp
addLedCheckBox("Sidebar on right",
		m_sideBarOnRight, SLOT(toggleSideBarOnRight(bool)));
```
Also, I suggest changing the text a bit. IMO it should be something like `Show sidebar on the right-hand side`.

Reply:
> As there's `addLedCheckBox` function, you can do it in two lines:

I realize now that I'm certain I started this from the `stable-1.2` branch instead of `master`. I must not have realized that when I rebased it.

> Also, I suggest changing the text a bit. IMO it should be something like `Show sidebar on the right-hand side`.

Sure thing! I just put it as a placeholder until I got a good suggestion of what would be preferred.